src/jobs/models/jobManager.js

```diff
@@ -65,17 +66,15 @@ module.exports.stopRun = async function (jobId, runId) {
 
 module.exports.deleteAllContainers = async function () {
     const configData = await configHandler.getConfig();
-    let runnerImage = configData.runner_docker_image;
     const jobConnector = require(`./${configData.job_platform.toLowerCase()}/jobConnector`);
-    let runnerImageWithoutTag = runnerImage.substring(0, runnerImage.indexOf(':'));
-    let result = await jobConnector.deleteAllContainers(runnerImageWithoutTag);
+    let result = await jobConnector.deleteAllContainers(PREDATOR_RUNNER_PREFIX);
```

Comment:
```suggestion
    let result = await jobConnector.deleteAllContainers(PREDATOR_RUNNER_JOB_APP_LABEL);
```

Reply:
the job manager is common for a all 3 paltform
so it's better not use the 'label'